cmd/terramate/cli/fmt.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2024 Terramate GmbH
+// SPDX-License-Identifier: MPL-2.0
+
+package cli
+
+import "fmt"
+
+func sprintf(format string, a ...interface{}) string {
+	return fmt.Sprintf(format, a...)
+}
```

Comment:
I don't think it makes sense a separate file just for this function. Also the name `fmt.go` is misleading, as we have a `terramate fmt` feature. You know, we do fuzzy file search when looking for the file that implements feature XYZ, then this will pop up very often but it's actually that kind of file that would be rarely open.

Reply:
that makes sense, I could move it to the bottom of `cmd/terramate/cli/cli.go`?